### ISSUE TITLE
Object files from src/ should end up in board-specific build directories

### ISSUE THREAD

**Issue by kbeckmann** [arturo182/tinyuf2#11]
When building for multiple boards, the object files from the shared sources `src/` end up in the same directory, i.e. `_build/src/`.

I gave it a shot and tried to fix it, but can't come up with a simple solution. Creating this issue for visibility. For now, one has to remove _build before building for a different board, otherwise the usb descriptor etc will be reused from the previous build.

**Comment by arturo182** [comment]
Yes, it's a tradeoff of using tinyusb's build system which we might outgrow at some point, no easy fix for that one right now.

**Comment by mubes** [comment]
For ecp5 I already took out the tinyusb build system because it was too inflexible. If you want I can make the same change for the other platforms which will actually simplify things for ecp5?  The changes I made already almost address this issue because otherwise you get namespace clashes (or at least the potential for them)....wouldn't be difficult to prepend the target name into the build output directory.

**Comment by arturo182** [comment]
Sure, go for it :)
